examples/vision/image_classification_from_scratch.py

```diff
@@ -61,16 +61,25 @@
 
 import os
 
+
+def is_jfif(path):
```

Comment:
Prefer in-lining this rather than creating a separate function for it, since it is only used once.

Reply:
Thanks for the comment. I moved it in-line. The order of instructions must be re-ordered comparing to the code in master as file must be closed before removal.